ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -2065,3 +2065,62 @@ def unique_consecutive(
         counts=ivy.array([2, 2, 1, 2, 1]))
     """
     return ivy.current_backend(x).unique_consecutive(x, axis=axis)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@to_native_arrays_and_back
+@handle_array_function
+def trim_zeros(
+    filt: Union[Sequence, ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    trim: str = "fb",
+) -> Union[ivy.Array, ivy.NativeArray]:
```

Comment:
As this function is primary and always return a single array, please add `out` argument [per the docs](https://unify.ai/docs/ivy/overview/deep_dive/inplace_updates.html#out-argument)

Reply:
Also please try to figure out if we need to add `@handle_view` and `copy` kwarg or not, as most manipulation functions have them, and the numpy docs is not clear whether it returns a view. For example, we added `@handle_view` to `ivy.vsplit` even though it's not even mentioned in the `numpy.vsplit` [docs](https://numpy.org/doc/stable/reference/generated/numpy.vsplit.html)

Reply:
I've added the `out` argument. It seems this function does not return a view based on numpy's docs. It also does not return an ivy function call that returns a view. so I removed `@handle_view`